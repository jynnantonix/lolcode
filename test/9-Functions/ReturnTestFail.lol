HAI,	BTW testing return statements

HOW DUZ I testReturn YR code
    BOTH SAEM code AN 1
    O RLY?
	YA RLY
	    FOUND YR 10
	MEBBE BOTH SAEM code AN 2
	    FOUND YR 20
	MEBBE BOTH SAEM code AN 3
	    FOUND YR 30
	NO WAI
	    FOUND YR NOOB
    OIC
IF U SAY SO

VISIBLE testReturn 1 MKAY? MKAY?
VISIBLE testReturn 2 MKAY? MKAY?
VISIBLE testReturn 3 MKAY? MKAY?
VISIBLE testReturn NOOB MKAY? MKAY?

FOUND YR NOOB

KTHXBAI