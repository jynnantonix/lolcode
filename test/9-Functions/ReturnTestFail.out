10 
20 
30 
None 
Unexpected error: Found return statement outside function.
